app/controllers/application_controller.rb

```diff
@@ -3,9 +3,16 @@ class ApplicationController < ActionController::Base
   protect_from_forgery
 
   before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
I think this should go inside the `Users::SessionsController` - I know we don't have this currently, but the idea would be to keep the ApplicationController generic/abstract  (as almost every controller is inheriting from it. And here we are reducing the abstraction by using the `if: :devise_controller` check). 

So instead, we could add a new controller `Users::SessionsController < Devise::SessionsController` which has 
`before_action :user_signup_params` (which then specifies which params are permitted).

For this to work, you would also need to add the following to routes.rb:
`devise_for :users, controllers: { sessions: 'users/sessions' }`

Reply:
I agree. I created `Users::RegistrationsController` (because it's about sign up only) and moved that stuff in it.